src/coreclr/vm/methodtablebuilder.cpp

```diff
@@ -10181,6 +10181,8 @@ void MethodTableBuilder::CheckForSystemTypes()
             // although applying 16 byte alignment is consistent with treatment of 128 bit SSE types
             // even on X86
             pLayout->m_ManagedLargestAlignmentRequirementOfAllMembers = 16; // sizeof(__int128)
+#elif defined(TARGET_WASM)
```

Comment:
This looks equivalent to the previous #elif, any reason for having it separate?

Reply:
Not really, it is just that I would need to update the comment and I am also not sure if it is the final value. I don't know yet if we will need 16 bytes alignment for v128. It looks like 4 might be enough and 8 for int128 which is represented as structure of 2x int64 in wasm.